benchmarks/build.gradle

```diff
@@ -38,31 +39,52 @@ dependencies {
             libraries.mockito,
             libraries.hdrhistogram,
             libraries.netty_tcnative,
-            libraries.netty_epoll
+            libraries.netty_epoll,
+            libraries.math
 }
 
 configureProtoCompilation()
 
+def vmArgs = [
+        "-server",
```

Comment:
@louiscryan All options except for heap size and `AggressiveOpts` are on by default. AFAIK `AggressiveOpts` enables experimental optimizations that haven't proven to be useful / stable enough to be enabled by default. I think we should reduce the VM flags to `-Xms2g -Xmx2g`. WDYT?